packages/eslint-plugin-react-hooks/src/index.js

```diff
@@ -18,9 +23,15 @@ export const configs = {
       'react-hooks/exhaustive-deps': 'warn',
     },
   },
-};
-
-export const rules = {
-  'rules-of-hooks': RulesOfHooks,
-  'exhaustive-deps': ExhaustiveDeps,
+  'flat/recommended': {
+    plugins: {
+      'react-hooks': {
+        rules,
+      },
+    },
+    rules: {
+      'react-hooks/rules-of-hooks': 'error',
+      'react-hooks/exhaustive-deps': 'warn',
+    },
```

Comment:
this is same in `recommended` and `flat/recommended`, how about assigning this to a const and reusing in both configs?